proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -44,157 +44,127 @@ direction of the policies set by the Board.
+
+Voting members are expected to make a time commitment which
+allows them to be responsive to CPC business, participate regularly
+in meetings and to participate in all voting matters (either by
+voting or specifically abstaining).  They are
+also expected to help to enable work of the regular CPC
+members by providing leadership, help with interactions with the
+board and Foundation staff and to generally help keep things moving.
+
+Voting members serve for a term of 1 year and must be re-nominated
+and ratified by the board each year. In addition, a Voting CPC member
+can be removed by voluntary resignation, by a standard CPC motion,
+or in accordance to the participation rules described for Regular
+members.
+ 
 Changes to CPC membership should be posted in the agenda, and may be suggested
 as any other agenda item.
 
-No more than one-fourth of the CPC members may be affiliated with the
-same employer. If removal or resignation of a CPC member, or a change of
-employment by a CPC member, creates a situation where more than
-one-fourth of the CPC membership shares an employer, then the situation
-must be immediately remedied by the resignation or removal of one or more
-PC
+No more than one-fourth of the Voting CPC members may be affiliated with the
+same employer. Removal or resignation of a Voting CPC member, or a change of
+employment by a Voting CPC member could cause more than
+one-fourth of the Voting CPC membership to be affiliated with the same emplyer.
+Such a situation must be immediately remedied by the resignation or
+removal of one or more voting CPC members.
 
 The CPC may, at its discretion, invite any number of non-voting observers to
 participate in the public portion of CPC discussions and meetings.
 
 The CPC shall meet regularly using tools that enable participation by the
-community. The meeting shall be directed by
-the CPC Chairperson. Responsibility for directing individual meetings may be
-delegated by the CPC Chairperson to any other CPC member. Minutes or an
-appropriate recording shall be taken and made available to the community
-through accessible public postings.
-
-\>\>\>\>\>\>\>\>\>\>\> Node.js
-
-CPC members are expected to regularly participate in CPC activities.
-In the case where an individual CPC member -- within any three month period --
-attends fewer than 25% of the regularly scheduled meetings, does not
-participate in CPC discussions, *and* does not participate in CPC votes, the
-member shall be automatically removed from the CPC. The member may be invited
-to continue attending CPC meetings as an observer.
-
-===========
+community. The meeting shall be directed by the CPC Chairperson.
+Responsibility for directing individual meetings may be
+delegated by the CPC Chairperson to any other CPC member. The CPC chairperson
+shall be one of the Voting members, and will be selected by the CPC members
+through consensus or if necessary a vote as described in the section titled "Voting".
```

Comment:
should the default be an election / vote?

Reply:
I'm open either way, but not sure why this decision would be different from others?  Maybe a vote is easier though, particularly if the vote is private.